doc/man3/OPENSSL_fork_prepare.pod

```diff
@@ -51,6 +51,11 @@ L<OPENSSL_init_crypto(3)>
 
 These functions were added in OpenSSL 1.1.1.
 
+=item *
+
```

Comment:
Add this text to the SYNOPSIS section just before the declarations of the functions (see numerous other man pages for an example):
````
Deprecated since OpenSSL 3.0, can be hidden entirely by defining
B<OPENSSL_API_COMPAT> with a suitable version value, see
L<openssl_user_macros(7)>:
````

We should also state in the description that they are deprecated and normally we would also say what applications should call instead. In this case it is not necessary for applications to call anything instead - so we should probably say that.